src/desktop_app_info.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use DesktopAppInfo;
+use libc;
+use ffi;
+use glib_ffi;
+use glib::translate::*;
+
+impl DesktopAppInfo {
+    pub fn search(search_string: &str) -> Vec<Vec<String>> {
```

Comment:
A double check on this function implementation would be very useful!

Reply:
Looks good for me

Reply:
:+1: